core-bundle/src/Crawl/Escargot/Factory.php

```diff
@@ -196,9 +196,6 @@ private function registerSubscribers(Escargot $escargot, array $selectedSubscrib
         }
     }
 
-    /**
-     * @throws \InvalidArgumentException
```

Comment:
`\InvalidArgumentException` needs to be removed from the other methods in this class too IMO.

Reply:
Changed in 4b170582f71bd8906902e2b5fd5ef2205966ae50.